types/smallwei__avue/global.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'vue' {
+    interface GlobalComponents {
+        AvueCrud: typeof AvueCrud;
+        AvueForm: typeof AvueForm;
+        AvueTree: typeof AvueTree;
+    }
+    interface ComponentCustomProperties {
+        $ImagePreview: (list: Array<{ url?: string; thumbUrl?: string }>, index?: number) => void;
+    }
+}
+
+export {};
```

Comment:
There is no 'global.js` file with a symbols, correct me if I'm wrong:
https://gitee.com/smallweigit/avue/tree/master/src
those TS symbols can be added to 'index.d.ts' file and re-imported in other files here